server/test/unit/apiTest/authorizeTest.js

```diff
@@ -6,28 +6,16 @@ const sinonChai = require('sinon-chai')
 
 // In-house dependencies
 const { helpers } = require('brave-alert-lib')
-
+const { mockResponse } = require('../../../testingHelpers')
 const api = require('../../../api')
 
 // Configure Chai
 use(sinonChai)
 
 const sandbox = sinon.createSandbox()
 
-function authorize_next() {
-  helpers.log('authorize_next')
-}
-
-function mockRequest(_req) {
-  const req = _req
-  return req
-}
```

Comment:
Haha, good point. This isn't really needed if it's just returning the same value as what was passed in, eh?